libs/akita-ng-effects/src/lib/types.ts

```diff
@@ -1,17 +1,22 @@
 import { Subject } from 'rxjs';
 
-
 export interface EffectOptions {
```

Comment:
Thanks. Few comments:

- Set `true` (recommended) 
It's not recommended. It's based on your use-case. 

- or omit if you want the effect to dispatch an action.
If it's omitted or `false`, it'll not dispatch an action.


- Do not forget to set `false` for the effects which are not intended to dispatch.
`false` is the default.

Reply:
This snippet makes `dispatch` to be `false` by default. I saw this snippet before I updated the comments.
https://github.com/datorama/akita/blob/e6f0b5d83684e9ad84ddbf065c6c571912b2c50c/libs/akita-ng-effects/src/lib/create-effect.util.ts#L9

I think you would like to see a documentation/comments as short as possible without redundant precautions. 
And of course without overseen mistakes.

```
This parameter dictates if the effect can dispatch an action. Set `true` if you want the effect to dispatch an action. Set to `false` or omit if the effect should not dispatch any action.
```

Is this formulation ok for you?

Reply:
LGTM

Reply:
The comment is now corrected